docs/commonbeam.rst

```diff
@@ -0,0 +1,22 @@
+.. _com_beam:
+
+Finding the smallest common beam
+================================
+
+radio-beam implements an exact solution for sets of 2 beams and an approximate method---`the Khachiyan algorithm <https://en.wikipedia.org/wiki/Ellipsoid_method>`_---for larger sets. The former case is straightforward to compute as the beams can be transformed into a space where the larger beam is circular to find the overlap area (`~radio_beam.commonbeam.common_2beams`). Our implementation borrows from the implementation in `CASA <https://casa.nrao.edu/>`_ (see `here <https://open-bitbucket.nrao.edu/projects/CASA/repos/casa/browse/code/imageanalysis/ImageAnalysis/CasaImageBeamSet.cc>`_).
```

Comment:
Can you comment on where the known failures in CASA are?  It does fail in some cases, right?

Reply:
:+1: